### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by alexxa**

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically.
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

File: README.md [hunk 1]
```diff
@@ -1,96 +1,64 @@
...
 
 ```bash
 $ git clone -b release-3.7 https://github.com/openshift/openshift-ansible
 ```
 
-## (Optional) Setup a cluster
-
-This section describes how to setup cluster on given machines. There are two
-playbook which you can use depends on what cluster you want to bring up.
+## Set up a cluster
```
Comment (fabiand) [inline README.md]: We should highlight that this is optional if you alreday have a cluster running.

File: README.md [hunk 2]
```diff
@@ -1,96 +1,64 @@
 # KubeVirt Ansible
 
-This repository provides collection of playbooks to
-- [x] [Install KubeVirt on existing OpenShift cluster](#install-kubevirt-on-existing-cluster)
-- [ ] Deploy Kubernetes cluster on given machines and install KubeVirt
-- [x] [Deploy OpenShift cluster on given machines and install KubeVirt](#deploy-kubernetes-or-openshift-and-kubevirt)
-- [x] [Provision resources, deploy cluster and install KubeVirt](#deploy-new-kubernetes-or-openshift-cluster-and-kubevirt-with-lago)
+This repository provides a collection of playbooks to
+- [x] [Install KubeVirt on an existing OpenShift cluster](#install-kubevirt-on-existing-cluster): install-kubevirt-on-openshift.yml
```
Comment (fabiand) [inline README.md]: Fine in adding the yamls at the end, but please enclose them in backticks: `…`

I'd actually also put them in bracets (`…`)

File: README.md [hunk 3]
```diff
@@ -188,10 +137,24 @@ $ ansible-playbook -i inventory \
...
 
+where
+* `cluster_type=kubernetes|openshift` defines a desired cluster type, 
+* `openshift_ansible_dir` is a path to a cloned [OpenShift Ansible][openshift-ansible-project] git repository,
+* `provider` defines provider. At the moment, `lago` is the only supported provider,
+* `inventory_file` is a path to the inventory file. Lago adds VMs into the inventory file once they are created, so it needs to know where it is located.
+
+## Questions ? Help ? Ideas ?
+
+Stop by the [#kubevirt](https://webchat.freenode.net/?channels=cnv-qe) chat channel on freenode IRC
```
Comment (lukas-bednar) [inline README.md]: s/cnv-qe/kubevirt/

File: README.md [hunk 4]
```diff
@@ -157,27 +114,19 @@ $ ansible-playbook -i localhost, --connection=local \
...
 
 
-## Deploy new Kubernetes or OpenShift cluster and KubeVirt with Lago
-
-This section describes how to deploy KubeVirt or OpenShift cluster with KubeVirt installed on it using Lago.
-
-### Parameters
+## Deploy a new Kubernetes or OpenShift cluster and KubeVirt with Lago
 
-| Name             |  Value        | Description                            |
```
Comment (lukas-bednar) [inline README.md]: I rather prefer table for parameters, but if ppl find that list form more readable let it be then.

File: README.md [hunk 5]
```diff
@@ -1,96 +1,61 @@
...
 
-This repository provides collection of playbooks to
-- [x] [Install KubeVirt on existing OpenShift cluster](#install-kubevirt-on-existing-cluster)
-- [ ] Deploy Kubernetes cluster on given machines and install KubeVirt
-- [x] [Deploy OpenShift cluster on given machines and install KubeVirt](#deploy-kubernetes-or-openshift-and-kubevirt)
-- [x] [Provision resources, deploy cluster and install KubeVirt](#deploy-new-kubernetes-or-openshift-cluster-and-kubevirt-with-lago)
+This repository provides a collection of playbooks to
+- [x] [Install KubeVirt on an existing OpenShift cluster](#install-kubevirt-on-existing-cluster): `install-kubevirt-on-openshift.yml`
+- [ ] Deploy a Kubernetes cluster on given machines and install KubeVirt: `deploy-kubernetes.yml`
+- [x] [Deploy an OpenShift cluster on given machines and install KubeVirt](#deploy-kubernetes-or-openshift-and-kubevirt): `deploy-openshift.yml`
```
Comment (lukas-bednar) [inline README.md]: `deploy-openshift.yml` playbook doesn't install kubevirt, it deploys openshift on given machines. you need to run `install-kubevirt-on-openshift.yml` to install it.

Comment (gbenhaim) [reply README.md]: @alexxa Any updates on this one?

File: README.md [hunk 6]
```diff
@@ -157,27 +110,19 @@ $ ansible-playbook -i localhost, --connection=local \
         kubevirt_mf=kubevirt.yaml" \
         install-kubevirt-on-openshift.yml
 ```
+where
+* `kubevirt_mf` is a path to KubeVirt manifest. See [releases](https://github.com/kubevirt/kubevirt/releases) or build it from sources,
+* `openshift_ansible_dir` is a path to a cloned [OpenShift Ansible][openshift-ansible-project] git repository,
+* `kconfig` is a path to the [`~/.kube/config`](https://docs.openshift.com/container-platform/3.7/cli_reference/manage_cli_profiles.html#switching-between-cli-profiles) file containing authentication information for the desired cluster.
```
Comment (gbenhaim) [inline README.md]: `kconfig` was changed to `kubeconfig` in https://github.com/kubevirt/kubevirt-ansible/pull/72

Comment (alexxa) [reply README.md]: @gbenhaim right, updated, thanks

File: README.md (hunk 5 above)
Comment (alexxa) [reply README.md]: @lukas-bednar @gbenhaim Fixed.

**Pull request merged**